lib/spack/docs/testing_guide.rst

```diff
@@ -0,0 +1,123 @@
+.. _testing-guide:
+
+=============
+Testing Guide
+=============
+
+This guide is intended for developers or administrators who want to
+integrate or test software releases built with Spack.
+
+---------
+Reporting
+---------
+The install and testing infrastructure of Spack supports multiple output formats for
+reporting the result of package builds in tools such as Jenkins or CDash.
+
+The output formats can be enabled by e.g.
+
+.. code-block:: console
+
+   $ spack install --log-format=junit <spec>
+
+Per default the log results will be placed into `var/<format>/test-<short_spec>.xml`.
```

Comment:
Single backticks make things italic, you probably want double backticks to make it monospace.